spec/summaryComponentSpec.js

```diff
@@ -0,0 +1,16 @@
+"use strict";
+
+var assert = new Assert();
+
+function hasUrl() {
+  var storyUrl = "http://www.telegraph.co.uk/fashion/news/why-going-barefoot-has-become-the-new-status-symbol/";
```

Comment:
Should we be using The Guardian newspaper links? Or was this deliberately a different paper to avoid confusion with tests?